app/views/admin/users/new.html.erb

```diff
@@ -45,7 +45,17 @@
                         <%= f.input_field :organization_id, collection: @organizations, class: "form-control" %>
                       <% end %>
                       <%= render 'admin/users/user_form_fields', f: f %>
-
+                      <!-- Only display password fields when not nested within organization form -->
+                      <% if @organization.nil? %>
+                        <%= f.input :password, label: "Password", wrapper: :input_group, required: true, hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length) do %>
+                          <span class="input-group-text"><%= fa_icon "lock" %></span>
+                          <%= f.input_field :password, class: "form-control" %>
+                        <% end %>
+                        <%= f.input :password_confirmation, label: "Confirm Password", wrapper: :input_group, required: true do %>
+                          <span class="input-group-text"><%= fa_icon "check" %></span>
+                          <%= f.input_field :password_confirmation, class: "form-control" %>
+                        <% end %>
+                      <% end %>
```

Comment:
I don't think superusers should have access to the password even here. Users should be able to manage their own passwords - if they can't, they're going to have other problems. @cielf what do you think?

Reply:
When we add a new user as a superuser, does it send out an invite?  If so, would they not be setting their password then?    [If not, that's a different question]

Reply:
I do know that I've been doing most of the support of this system for about a year,  and I've never set a users password for them.